src/pad.ts

```diff
@@ -28,8 +22,8 @@ const utils = require('./utils')
  * @param {NamedNode} author - The author of text being displayed
  * @returns {String} The CSS color generated, constrained to be light for a background color
  */
-UI.pad.lightColorHash = function (author) {
```

Comment:
I still need to add the types to the parameters.  Just realized as there were so many other things to change. I'll have you look at my comments first and after our meeting I'll do the types in the parameters.